src/styles/GlobalStyle.js

```diff
@@ -155,6 +155,10 @@ const GlobalStyle = createGlobalStyle`
   .dropdown-menu {
     padding: 0;
   }
+  
+  .public-DraftEditor-content {
+    overflow-wrap: anywhere !important;
```

Comment:
Is the `!important` necessary?

Reply:
Yes cause the value I'm overriding is applied through inline attributes by Draft so that was the only way I found to get higher specificity :/ Couldn't find a prop to modify the attributes it sets directly